pkgs/applications/networking/instant-messengers/jitsi-meet-electron/default.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv
+, fetchurl
+, appimageTools
+, makeWrapper
+, electron_8
+}:
+
+stdenv.mkDerivation rec {
+  pname = "jitsi-meet-electron";
+  version = "2.0.0";
+
+  src = fetchurl {
+    url = "https://github.com/jitsi/jitsi-meet-electron/releases/download/v${version}/jitsi-meet-x86_64.AppImage";
+    sha256 = "11ci9dqhy8hkb4fwykjvcvai20ahqhjil825n1y1xf663ch8by93";
+    name="${pname}-${version}.AppImage";
+  };
+
+  appimageContents = appimageTools.extractType2 {
+    name = "${pname}-${version}";
+    inherit src;
+  };
+
+  dontUnpack = true;
+  dontConfigure = true;
+  dontBuild = true;
+  dontPatchELF = true;
```

Comment:
are we sure we need this? I believe this was needed in some expressions because it used `runtimeDependencies` in `autoPatchelfHook`.

Reply:
removed in 91aa477ee776a7c5de7a28fcc2922d888d14c684